examples/wave_tank/ex1/RigidBodyKinematics.cpp

```diff
@@ -60,6 +27,9 @@
 // IBTK THIRD-PARTY INCLUDES
 #include "muParser.h"
 
+// APPLICATION INCLUDES
+#include "RigidBodyKinematics.h"
```

Comment:
👍 

These separated sections are leftover from the dark days before clang-format could readily keep headers grouped together automatically.

Reply:
It would be nice to clean all this up, but ultimately since SAMRAI 2 doesn't put all its `#include`s in a folder we don't have a good way to distinguish between `PatchLevel.h` and `RigidBodyKinematics.h`. Hopefully this is fixed in newer versions.